src/rust/engine/fs/src/snapshot.rs

```diff
@@ -805,19 +837,23 @@ mod tests {
     )
   }
 
-  fn expand_all_sorted(posix_fs: Arc<PosixFS>) -> Vec<PathStat> {
-    let mut v = posix_fs
-      .expand(
-        // Don't error or warn if there are no paths matched -- that is a valid state.
-        PathGlobs::create(
-          &["**".to_owned()],
-          &[],
-          StrictGlobMatching::Ignore,
-          GlobExpansionConjunction::AllMatch,
-        )
-        .unwrap(),
+  fn expand_all_sorted(
+    posix_fs: Arc<PosixFS>,
+    runtime: &mut tokio::runtime::Runtime,
```

Comment:
Thoughts on this being explicit vs being thread local? How do you think that we should think about that boundary?

I'm fine either way: is easy to tweak later.

Reply:
It turns out the blocking stuff can't happen on a current_thread runtime, so we need an explicit non-current_thread runtime here. We could make a method-local one rather than passing one in, but I suspect that would just be a little harder to follow :)

Reply:
...oh! Have we actually made all usage of the `Runtime` explicit in the other cases as well? That's pretty awesome... I guess I missed the uppercase-static vs lowercase-local distinction. #shipit

Reply:
We'll find out if any tests start failing!